### PR TITLE
[DOCFIX] Add documentation about thrift framesize problem

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Debugging-Guide.md [hunk 1]
```diff
@@ -55,7 +55,22 @@ Typical issues:
...
+
+### Q: I'm seeing error messages like "java.io.IOException: Frame size (67108864) larger than max length (16777216)". What is wrong?
+
+A: This problem can be caused by different possible reasons.
+
+- Please double-check if Alluxio master address has the correct port specified. The default listening port for master is port 19998,
+while a common mistake when this error is seen is to use a wrong port 19999---the default the Web UI port for Alluxio master. 
+- Please make sure that security setting on Alluxio client and master sides are consistent. 
+Alluxio provides different [authentications](Security.html#authentication).
+This error happens if one side is configured to have `alluxio.security.authentication.type` to `SIMPLE`,
```
Comment (gpang) [inline docs/en/Debugging-Guide.md]: Is this the only situation where this happens? What if one is SIMPLE and the other is CUSTOM? Is it just "mismatched settings", or do only some combinations cause this error?

**Comment by gpang** [comment]
@apc999 Thanks, I left a minor question.

File: docs/en/Debugging-Guide.md (hunk 1 above)
Comment (apc999) [inline docs/en/Debugging-Guide.md]: I meant "mismatched setting"
text udpated

**Comment by apc999** [comment]
@gpang PTAL

**Pull request merged**